### ISSUE TITLE
Feature flags not available on custom vitals

### ISSUE THREAD

**Issue by pg-synth** [DataDog/browser-sdk#3076]
Hello,

It's great that we can now collect custom vital with RUMs, it really allows us to measure more precisely custom performances in our FE application.

We are using feature flags in order to test performances updates..thing is.. feature flags are not attached to vitals by default.. so we can't group by a given FF in order to see how it's impacting the vital duration in RUM.

We hack our way through by passing a given FF and its evaluated value as global context that is attached to vitals by default.. but having FF on vitals OOTB would truly be useful. 

That's all, have a great day.

**Comment by erykpiast** [comment]
Yes, please! I was actually very surprised when I learned some time ago flags aren't available for Vitals. It feels natural they'd be, same as for other measures.

**Comment by BenoitZugmeyer** [comment]
Hello, thank you for your feedback. This is under our radar: we want to add feature flags on more events. Although, having a lot of feature flags means bigger events, and bigger events often means degraded reliability (because we can't send as much data on lower connections for example). So we want to be careful about it.

We are tracking this issue on our side and will get back to you when we have something new.
